src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -359,10 +400,33 @@ private void markAsRead() {
             issue.markAsRead(UI.prefs);
 
             parentPanelControl.refresh();
+            shouldTriggerIssueSelectedEvent = true;
             listView.selectNextItem();
         }
     }
 
+    //Mark all issues below and including the selected issue as unread.
+    private void markAllBelowAsUnread() {
+        if (listView.getSelectedIndex() >= 0) {
+            for (int i = listView.getSelectedIndex(); i < listView.getItems().size(); i++){
+                TurboIssue issue = listView.getItems().get(i).getIssue();
+                issue.markAsUnread(UI.prefs);
+            }
+            parentPanelControl.refresh();
```

Comment:
This is necessary. The bold text for unread issues will not appear unless we do a quick refresh.